### ISSUE TITLE
ejs code beautify failure

### ISSUE THREAD

**Issue by TOP-Chao** [Glavin001/atom-beautify#806]
```html
<!DOCTYPE html>
<html>
    <head>
        <meta charset="utf-8">
        <title></title>
    </head>
    <body></body>
    <script src="/socket.io/socket.io.js"></script>
    <script>
        var socket = io("<%= path %>");
        socket.on('message', function(message, next) {
            console.log(message);
            next();
        });
        socket.emit('subscribe', {
            'uid': "<%= uid %>",
            'room': [1]
        });
    </script>
</html>
```
`ctr-alt-b` 
```html
<!DOCTYPE html>
<html>
    <head>
        <meta charset="utf-8">
        <title></title>
    </head>
    <body></body>
    <script src="/socket.io/socket.io.js"></script>
    <script>
        var socket = io("<%= path %>);
        socket.on('message', function(message, next) {
            console.log(message);
            next();
        });
        socket.emit('subscribe', {
            'uid': "<%= uid %>,
            'room': [1]
        });
    </script>
</html>
```

**Comment by prettydiff** [comment]
I can reproduce this defect on Pretty Diff.  Two missing quotes.

**Comment by prettydiff** [comment]
Fixed in Pretty Diff v1.16.16.  The corrected code is at http://prettydiff.com/lib/jspretty.js

Until a new version of Atom Beautify is released you will need to update the Pretty Diff dependency to the latest version.

* You can copy the corrected file over the current file in your local distribution.
* You can also uninstall / reinstall Atom Beautify to get the latest dependencies
